pkg/gui/view_helpers.go

```diff
@@ -12,9 +12,9 @@ import (
 
 func (gui *Gui) refreshSidePanels(g *gocui.Gui) error {
 	// not refreshing containers and services here given that we do it every few milliseconds anyway
-	if err := gui.refreshImages(); err != nil {
-		return err
-	}
+	// if err := gui.refreshImages(); err != nil {
```

Comment:
I would keep this here. We call refreshSidePanels upon performing some action e.g. removing an image. We'd want that change to be reflected in the UI immediately.

Reply:
Method `refreshSidePanels` is only called in `loadNewDirectory` which in turn gets called only once 

In the current code base to refresh the image section after removing/pruning an image method `gui.refreshImages()` gets called 

Since now `gui.refreshImages()` gets called on a timer `refreshSidePanels` basically does not do anything